src/nvim/popupmnu.c

```diff
@@ -219,7 +220,9 @@ void pum_display(pumitem_T *array, int size, int selected)
     if (curwin->w_p_rl) {
       pum_width = pum_col - pum_scrollbar + 1;
     } else {
-      pum_width = Columns - pum_col - pum_scrollbar;
+      assert(Columns - pum_col - pum_scrollbar >= INT_MIN
```

Comment:
I haven't done it yet because I'm looking for the optimal "shape" for those macros. Not so easy as it can seem...